### PR TITLE
Add cropped images, when aspect ration is < 0.6

### PR DESCRIPTION
**Pull request by pratyush-hub**

### CODE REVIEW

File: Processor/processor.py [hunk 1]
```diff
@@ -5,10 +5,50 @@
...
+        width, height = img.size
+        return width / height < 0.06
+
+    @staticmethod
+    def split_images(img, ext):
+        width, height = img.size
+        chunks = math.ceil(0.06 / (width / height))
+        chopsize = math.ceil(height / chunks)
+        beg = """{
+                        "@context": {
```
Comment (nitsshukla) [inline Processor/processor.py]: we can change this to 

```
MEDIA_GALLERY_JSON_LD_TEMPLATE = '{\"@type\": \"g:MediaGallery\",\"g:layoutHint\": \"MediaGalleryLayout\",' \
                                     '\"s:image\": [' \

```

File: Processor/processor.py [hunk 2]
```diff
@@ -5,10 +5,50 @@
...
+import math
+import random
 
 
 class Processor:
 
+    @staticmethod
+    def aspect_ration_check(img):
+        width, height = img.size
+        return width / height < 0.06
```
Comment (nitsshukla) [inline Processor/processor.py]: Will move it to constants `ASPECT_RATIO_THRESHOLD = 0.06`

File: Processor/processor.py [hunk 3]
```diff
@@ -29,7 +56,12 @@ def __process_image_tag(self, tag):
             if tag.has_attr('ec-data-src'):
                 src = tag['ec-data-src']
         if src:
-            self.body += Constants.IMG_JSON_LD_TEMPLATE \
+            if CommonUtility.is_url(src):
+                img, ext = CommonUtility.read_image(src)
+                if self.aspect_ration_check(img):
```
Comment (nitsshukla) [inline Processor/processor.py:61]: Else condition missed, where image isn't above aspect ration, and yet it will not be part of json-ld.

**Comment by nitsshukla1** [comment]
Closing this, since we are no longer pursuing this path.

**Pull request closed without merging**